_posts/2021-02-12-generalist-specialization-and-the-impostor-syndrome.md

```diff
@@ -0,0 +1,80 @@
+
+So, while specialization is still rather a valid thing to do, you should also invest in learning a wide array of skills for software crafting, as listed above. These days you definitely need to understand automated testing, delivery, security, and cloud platforms, in addition to Frontend, Backend, and databases. But you can still specialize, you can double on on one or two things that bring you pleasure. You can actively practice them, learn more about them, and be the go-to-person for said techniques.
+
+But there's also another route. I'm a Java coder. That's been my forte. I started it in the 1990s, then I did a long period of teaching it for money, which meant I needed to first master it myself, and keep up to date. I would say Java is something I can code in my sleep, and sometimes do, so that's definitely my specialization and strength. But would you believe it if I said I have barely used any Java for several years now? I've been relying on my 25 years of experience on all things, and my ability to rapidly pick up new things based on that. If I drew my skills profile right now it would be crazy. I can do many things. Some might argue even any things. My recent years have been filled for example by Splunk and Excel coding, a lot of Python, Scala, some Kotlin, Gitlab pipelines, and lots and lots of cloud acronyms. Did you know that AWS has hundreds of services and thousands of acronyms? And each one is a specialization in itself.
+
+So I would call myself a generalist. That's been my specialization. Meaning that I actively pursue that and keep my generalization up to date. I know few things really in-depth, but I know a hundred things on a level that I can rapidly refresh up to today's requirements if there's a need. A few years back someone asked me about XSLT templates, so I whipped up from my ancient technologies archive a nice recursion template that lets you do crazy stuff with minimal code. That gives me much flexibility to jump in many places. Unfortunately, that comes with a downside too. A constant visitor.
+
+## How can you deal with the Impostor Syndrome?
+
+There are two bad ways to try to deal with this. One is to never put yourself in a position of change or challenge. Just play it very safe. You learned Java 1.0 in 1996, so keep on using it for everything, and only look for jobs where that is all that is required. This approach would let the Impostor Syndrome rule and limit you. Another is to wait, it will pass. But that's not going to help you deal with it any better next time, so while it may be a victory, it's a lame one.
```

Comment:
"with this" -> "with it" or "with impostor syndrome".

"…let impostor syndrome rule…"

"another strategy is to wait"